tests/testthat/_snaps/methods.md

```diff
@@ -0,0 +1,108 @@
+# print.lint, print.lints support optional message wrapping : width = 10
+
+    Code
+      print(lints, width = width)
```

Comment:
TBH, I like the parametrized snapshot tests better here.

The snapshot description would have been an issue if the title had been too long because then the varying info at the end if easy to miss, but it's not.